scripts/scripts.js

```diff
@@ -12,6 +12,14 @@
 
 import { setLibs, decorateArea } from './utils.js';
 import { captureProfile } from '../utils/event-apis.js';
+import BlockMediator from '../deps/block-mediator.min.js';
+
+/**
+ * Moving mediator up since it is not loading in time for autolinks
+ * to update. Should very litle to non visibile impact on the
```

Comment:
Let's remove this comments. I was told by some Milo devs even these few bytes should be saved, especially because they are before LCP.

Reply:
oh snap!, i forgot we don't clear this for prod! My bad!